### PR TITLE
Few Fixes to a few issues

### PR DESCRIPTION
**Pull request by Mayegelt**
#3301 Golden Keshig Power Lances
#3123 ZM Reactions updated
#3128 Daemons list with LoW Warlord not working
#3112 Venerable Dreads Legion Options added

### CODE REVIEW

**Comment by Mayegelt** [comment]
I'm going to try to do a bunch of fixes over this week if I can motivate myself. Most are really small and should be able to be pushed direct to main.

File: 2022 - Horus Heresy.gst [hunk 1]
```diff
@@ -1156,35 +1156,35 @@ In addition, when a Fast Vehicle moves, it may choose to move at Flat-out:</desc
...
+            <characteristic name="Description" typeId="c627-4637-8de5-65fb">When making Reactions during a Zone Mortalis battle, an army may not choose to use any of the Core Reactions presented in the Horus Heresy ~ Age of Darkness rulebook nor any army list specific Advanced Reactions
 Instead, all armies, regardless of Faction or size, may only choose to use any or all of the Zone Mortalis Reactions as presented in this supplement during a Zone Mortalis battle or any of the standard Advanced Reactions in the Horus Heresy - Age of Darkness rulebook.
 
-Base Raction Allotment, Additional Reactions and Reaction Limited in Smaller Games.
-In games with a total army size of 1,000 points or fewer per player, any player may make one (1) Reaction in each Phase of their opponent&apos;s turn. This number may not be increased in any way. Regardless of any modifications to a player&apos;s Reaction Allotment or special rules granting additional Reactions, the Reactive player may never make more than one(1) Reaction in any single Phase.</characteristic>
+Base Rection Allotment, Additional Reactions and Reaction Limited in Smaller Games.
```
Comment (nstephenh) [inline 2022 - Horus Heresy.gst:1162]: close but still not "reaction"

Comment (Mayegelt) [reply 2022 - Horus Heresy.gst:1162]: Fixed now

**Comment by Mayegelt** [comment]
https://github.com/BSData/horus-heresy/issues/3296

https://github.com/BSData/horus-heresy/issues/3297

https://github.com/BSData/horus-heresy/issues/3298

https://github.com/BSData/horus-heresy/issues/3299

https://github.com/BSData/horus-heresy/issues/3300

https://github.com/BSData/horus-heresy/issues/3301

https://github.com/BSData/horus-heresy/issues/3304

**Pull request merged**